Grove>> Grove>> Grove>> Traceback (most recent call last):
  File "grove.py", line 19, in <module>
    main()
  File "grove.py", line 9, in main
    x = Command.parse(s).eval()
  File "/mnt/c/Users/BrowderKD20/Grove/Grove/grove_lang_lang.py", line 127, in eval
    context[self.name.name] = self.value.eval()
  File "/mnt/c/Users/BrowderKD20/Grove/Grove/grove_lang_lang.py", line 310, in eval
    self.methodName(el)
TypeError: 'Name' object is not callable
